elasticapm/transport/http.py

```diff
@@ -70,19 +70,22 @@ def __init__(self, url: str, *args, **kwargs) -> None:
         self._http = None
         self._url = url
 
-    def send(self, data, forced_flush=False):
+    def send(self, data, forced_flush=False, custom_url=None, override_headers=None):
```

Comment:
Is there a reason to use `custom` for the URL override, and `override` for the custom headers? :D

Reply:
Haha, good catch. It went through a lot of iterations over the PR. I'll clean them up.